### DISCUSSION TITLE
Getting search engines to pick up my Gitbook site: SEO?

### DISCUSSION THREAD

**Discussion by byteasylum** [GitbookIO/community#797] [2. Published content]
Hello everyone, I created a new Gitbook website (currently free tier), and started writing some new articles in it.

I also experimented with setting up a new Wordpress site, on wordpress.com, but didn't add the same content to it.

After about a week, I noticed that my Wordpress site is easily showing up in search results on Google. However, the Gitbook site, which has much more unique content, isn't showing up in search results.

Because of how Gitbook is designed, I can't add / verify my Gitbook site in Google Search Console.

**Question**: How do I get Gitbook to pick up my website and start indexing it?

I **really** don't want to have to use Wordpress. Gitbook is so much nicer.

**Comment by nnachtegael** [comment]
same for me
I can't add a DNS TXT record if a CNAME record already exists

**Reply by gregberge** [reply]
Which cloud provider are you using? It should be supported. I recommend reaching out to them to clarify why it’s not working.

**Reply by nnachtegael** [reply]
my provider is ionos.fr

**Comment by rroihans** [comment]
Yeah me too, it is been weeks and there aren't any new list of my page being shown in the google after the last indexed article.

**Comment by wforte88** [comment]
Same problem here. Including CNAME and TXT records for the same domain is blocked according to DNS specifications (RFC 1034). Some domain services have an alternative method of declaring the record with a flag, but this isn't a global solution.

I have the Gitbook Ultimate plan and I can't capture data to optimize SEO, which is frustrating.

I understand that Gitbook has native SEO optimizations, but it doesn't allow you to see how the site is performing on Google; this is the differentiating factor of Google Search Console. This data is vital for understanding user behavior before they reach my Gitbook space. Gitbook's insights data only shows behavior after the user has accessed the pages, similar to Google Analytics.

Google Search Console has several verification methods, but none of them are supported by Gitbook:

**Uploading an HTML file to the root**
This is not possible in Gitbook.

**Adding a meta tag to the head of pages**
Gitbook doesn't allow customizing the head of pages.

**Native Google Analytics Javascript**
Gitbook has a plugin for Google Analytics, but it doesn't use GA's native analytics.js, so this validation method doesn't work either.

**Google Tag Manager**
Gitbook doesn't have a header editor to insert the tag, nor an integration plugin.

**TXT record**
Since Gitbook already uses a CNAME record for a space with a custom domain, several DNS management services will not allow the declaration of a TXT record for the same domain.

With all this, the question is: does Gitbook consider that integration with Search Console generates value for platform users, or is this outside the roadmap?
